plugins/videobasedtracker/UVCImageSource.cpp

```diff
@@ -219,6 +229,7 @@ namespace vbtracker {
         // Convert the image to at cv::Mat
         color = cv::Mat(rgb->height, rgb->width, CV_8UC3, rgb->data).clone();
 
+        uvc_free_frame(current_frame);
```

Comment:
could this perhaps be handled by a RAII "finally" from `<osvr/Util/Finally.h>`, so we don't accidentally leak frames?

(Or, tbh, I know it's @godbyk 's initial code, but could really go for using `unique_ptr` with custom deleter - yes, those exist! just need to be in the type - on those UVC frames and a little helper `make_uvc_frame_buffer` function that returns a buffer allocated with libuvc and managed by such a `unique_ptr`. That would be the ideal way to go.)